js/src/api/transport/ws/ws.js

```diff
@@ -131,6 +131,8 @@ export default class Ws extends JsonRpcBase {
     this._autoConnect = true;
     this._retries = 0;
 
+    this.emit('open');
```

Comment:
Only ws? We have 2 transports, needs to be in-sync.

Additionally, since it is done on the API layer, we need to have tests that prove that is does what it is supposed to for all the transports.